src/garage/sampler/__init__.py

```diff
@@ -18,6 +18,10 @@
 from garage.sampler.worker import Worker
 from garage.sampler.worker_factory import WorkerFactory
 
+# ISSampler requires TensorFlow
+if 'tensorflow' in sys.modules:
+    from garage.sampler.is_sampler import ISSampler
+
```

Comment:
If ISSampler requires TensorFlow, shouldn't `__all__ = [...]` be adjusted accordingly? Right now `'ISSampler'` is in `__all__` regardless of whether TensorFlow is available

Reply:
Great catch, updating that now.